widgetastic_manageiq/__init__.py

```diff
@@ -1437,6 +1437,24 @@ class DialogBootstrapSwitch(BootstrapSwitch):
     )
 
 
+class V2VBootstrapSwitch(BootstrapSwitch):
+    """locator in v2v is different than BootstrapSwitch"""
+
+    ROOT = ParametrizedLocator(
+        ".//div[./preceding-sibling::label[normalize-space(.)={@label|quote}]]"
+        '/div/div[contains(@class, "bootstrap-switch-container")]'
+    )
+
+    def fill(self, value):
```

Comment:
Question: Why overwrite `BootstrapSwitch`'s fill method here? As far as I can see, the only difference is that you're leaving out
```python,
            if self.selected != value:
                raise WidgetOperationFailed(
                    'Failed to set the bootstrap switch to requested value.')
```

Reply:
This Bootstrapswitch does not have the selected element (locator) hence it cannot neet this criteria of fill of standard BootstrapSwitch.

Reply:
The need to override default fill is still not clear to me here. You're still using `self.selected`, before filling. Is this property not returning a reliable value after the widget is clicked?

Reply:
![Screenshot from 2019-06-17 14-32-50](https://user-images.githubusercontent.com/3866976/59638337-fce4d100-910c-11e9-945b-ee6eafe9bde6.png)


The Bootstrapswitch is missing classes that can show the selected value as in attached ss

Reply:
![Screenshot from 2019-06-17 14-33-32](https://user-images.githubusercontent.com/3866976/59638383-200f8080-910d-11e9-9ae8-81e4ec8d9e47.png)

This is bootstrapswitch that has the selected element (classes with ng-empty and not empty as in https://github.com/RedHatQE/widgetastic.patternfly/blob/master/src/widgetastic_patternfly/__init__.py#L1812)